packages/fast-usdc-deploy/src/reimburse-opco.build.js

```diff
@@ -0,0 +1,63 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { AmountMath } from '@agoric/ertp';
+import { multiplyBy, parseRatio } from '@agoric/ertp/src/ratio.js';
+import { Far } from '@endo/far';
+import { parseArgs } from 'node:util';
+import { getManifestForReimburseOpCo } from './reimburse-opco.core.js';
+import { toExternalConfig } from './utils/config-marshal.js';
+
+/**
+ * @import {CoreEvalBuilder, DeployScriptFunction} from '@agoric/deploy-script-support/src/externalTypes.js'
+ * @import {ReimbursementTerms} from './reimburse-opco.core.js'
+ */
+
+const usage = 'Use: --destinationAddress <address> --principal <amount>';
```

Comment:
I wonder how/when we fill in these options...
when: I suppose in a "send core eval proposal to mainnet" issue.
how: I prefer that before https://github.com/Agoric/agoric-private/issues/307 is closed, the amount and address are added there.